app/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html5>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Alina's Band Name Generator</title>
+    <!-- Custom Stylesheet -->
```

Comment:
Cool! Can also put this in a separate file and include it here, much like you do the script.js file.